Extractor/NodeSources/NodeSetSource.cs

```diff
@@ -136,7 +142,7 @@ private void LoadReferences()
                     {
                         references[targetId] = targetRefs = new Dictionary<(NodeId, NodeId, bool), IReference>();
                     }
-                    if (!targetRefs.ContainsKey((node.NodeId, reference.ReferenceTypeId, reference.IsInverse)))
+                    if (!targetRefs.ContainsKey((node.NodeId, reference.ReferenceTypeId, !reference.IsInverse)))
```

Comment:
The flag here seems to have flipped, but I could not see an obvious change upstream of where it comes from. Raising it to confirm.

Reply:
It was a bug, actually. This feature never worked as it should... It is pretty niche.